package.json

```diff
@@ -76,12 +76,13 @@
         "babel-preset-stage-2": "^6.24.1",
         "babel-preset-vue": "^2.0.2",
         "cross-env": "^5.1.6",
-        "eslint": "^4.19.1",
+        "eslint": "^5.16.0",
+        "eslint-config-airbnb-base": "^13.1.0",
         "eslint-friendly-formatter": "^4.0.1",
         "eslint-loader": "^2.1.2",
-        "eslint-plugin-import": "^2.16.0",
-        "eslint-plugin-promise": "3.8.0",
-        "eslint-plugin-vue": "^4.5.0",
+        "eslint-plugin-import": "^2.17.2",
+        "eslint-plugin-promise": "4.1.1",
```

Comment:
Does Airbnb need this? We don't seem to use any of these rules ourselves.

Reply:
I could uninstall the promise plugin, import is still required